exist-core/src/test/xquery/xquery3/bang.xql

```diff
@@ -163,11 +163,16 @@ function bang:constructor() {
     $bang:works/employee[pnum = "P4"] ! <name>{@name/string()}</name>
 };
 
+(:~
+ : implicit context cannot be tested in a xquery testsuite
+ : because it will raise a static error now
```

Comment:
is there a ticket to trace this?

Reply:
I do not understand. What should be traced?

Reply:
I don't think this should be pending. If it is invalid it should be removed, or if it has a purpose it should be refactored so that it passes whilst still testing the subject.

Reply:
I will add this test to the Java test suite.